src/basic/missing.h

```diff
@@ -1011,6 +1011,10 @@ typedef int32_t key_serial_t;
 #define ETHERTYPE_LLDP 0x88cc
 #endif
 
+#ifndef EXTRA_CMSG_SPACE
+#define EXTRA_CMSG_SPACE 1024
+#endif
```

Comment:
This EXTRA_CMSG_SPACE thing is an ugly hack. We should figure out how much space is really needed any why, and not use 1kb blindly. But this is a subject for a different time. So please just move EXTRA_CMSG_SPACE to some better place (src/basic/socket-util.h ?).